client/components/wcc-settings-form/settings-group.js

```diff
@@ -98,6 +104,34 @@ const SettingsGroup = ( props ) => {
 				</div>
 			);
 
+		case 'summary':
+			const renderStepSummary = ( step, stepIndex ) => (
+				<div key={ stepIndex } className="settings-step-summary">
+					<h4>{ step.tab_title }</h4>
+					<Summary
+						formValues={ form.values }
+						layoutItems={ step.items }
+						summaryTemplate={ step.summary }
+						storeOptions={ storeOptions }
+						fieldsOptions={ form.fieldsOptions } />
+					<Button compact
+							onClick={ () => formActions.goToStep( stepIndex ) } >
+						{ __( 'Edit' ) }
+					</Button>
+				</div>
+			);
+
+			return (
+				<div className="settings-group-default">
+					<FormSectionHeading dangerouslySetInnerHTML={ sanitizeHTML( group.title ) } />
```

Comment:
Do we actually need to use `dangerouslySetInnerHTML` here?

Does `<FormSectionHeading>{ sanitizeHTML( group.title ) }</FormSectionHeading>` suffice?